utils_cv/tracking/model.py

```diff
@@ -282,46 +242,46 @@ def save(self, path) -> None:
         save_model(path, self.epoch, self.model, self.optimizer)
         print(f"Model saved to {path}")
 
-    def evaluate(self,
-                 results: Dict[int, List[TrackingBbox]],
-                 gt_root_path: str) -> str:
-        
+    def evaluate(
+        self, results: Dict[int, List[TrackingBbox]], gt_root_path: str
+    ) -> str:
+
         """ eval code that calls on 'motmetrics' package in referenced FairMOT script, to produce MOT metrics on inference, given ground-truth.
         Args:
             results: prediction results from predict() function, i.e. Dict[int, List[TrackingBbox]] 
             gt_root_path: path of dataset containing GT annotations in MOTchallenge format (xywh)
         Returns:
             strsummary: str output by method in 'motmetrics' package, containing metrics scores        
         """
-       
-        #Implementation inspired from code found here: https://github.com/ifzhang/FairMOT/blob/master/src/track.py
+
+        # Implementation inspired from code found here: https://github.com/ifzhang/FairMOT/blob/master/src/track.py
         evaluator = Evaluator(gt_root_path, "single_vid", "mot")
-        
+
         with tempfile.TemporaryDirectory() as tmpdir1:
-            os.makedirs(osp.join(tmpdir1,'results'))
-            result_filename = osp.join(tmpdir1,'results', 'results.txt')
-          
-            # Save results im MOT format for evaluation            
-            bboxes_mot = boxes_to_mot(results)            
+            os.makedirs(osp.join(tmpdir1, "results"))
+            result_filename = osp.join(tmpdir1, "results", "results.txt")
+
+            # Save results im MOT format for evaluation
+            bboxes_mot = boxes_to_mot(results)
             np.savetxt(result_filename, bboxes_mot, delimiter=",", fmt="%s")
 
             # Run evaluation using pymotmetrics package
-            accs=[evaluator.eval_file(result_filename)]
-                           
+            accs = [evaluator.eval_file(result_filename)]
+
         # get summary
         metrics = mm.metrics.motchallenge_metrics
         mh = mm.metrics.create()
-       
+
         summary = Evaluator.get_summary(accs, ("single_vid",), metrics)
         strsummary = mm.io.render_summary(
             summary,
             formatters=mh.formatters,
-            namemap=mm.io.motchallenge_metric_names
+            namemap=mm.io.motchallenge_metric_names,
         )
-        print(strsummary)        
-        
+        print(strsummary)
```

Comment:
[minor] Ideally this print shouldn't be here but in the notebook.

Reply:
Ok, Liqun fixed that in her PR.